api/server.py

```diff
@@ -0,0 +1,54 @@
+"""Script to start the server for Stampify"""
+
+import logging
+import uuid
+
+from flask import Flask, redirect, render_template, request, session, url_for
+
+from error import Error
+from stampifier import Stampifier
+
+app = Flask(__name__, static_folder='assets/')
+app.secret_key = uuid.uuid4().hex
```

Comment:
Why should it be random?

Reply:
We can have fixed string also, as we are not working with sensitive information of user. Using random has no reason in particular.

Reply:
Should it be const for everyone?

Reply:
Yes, I suppose.

Reply:
Updated to retrieve the key from environment variable in new changes.